settings/Core.setting.php

```diff
@@ -1209,7 +1209,7 @@
     'pseudoconstant' => array(
       'callback' => 'call://themes/getAvailable',
     ),
-    'default' => 'default',
+    'default' => 'minetta',
```

Comment:
If we really think that the theme setting should be explicit, then maybe this should be 'none'?